backend/committed_graphql_schema.graphql

```diff
@@ -680,6 +680,18 @@ type CommissionRates {
   bakingCommission: Decimal!
 }
 
+type Contract {
+  id: ID!
+  contractAddress: ContractAddress!
+  moduleRef: String!
+  balance: Long!
+  firstTransactionId: Long!
+  lastTransactionId: Long!
```

Comment:
I am not sure about this. Does this mean we are exposing internal ids in the API? Why are we not exposing hashes?

Reply:
ignored